spec/install/gems/compact_index_spec.rb

```diff
@@ -768,6 +768,27 @@ def start
     end
   end
 
+  it "performs partial update with a non-empty range" do
+    gemfile <<-G
+      source "#{source_uri}"
+      gem 'rack', '0.9.1'
+    G
+
+    # Initial install creates the cached versions file
+    bundle! :install, :artifice => "compact_index"
+
+    # Update the Gemfile so we can check subsequent install was successful
+    gemfile <<-G
+      source "#{source_uri}"
+      gem 'rack', '1.0.0'
+    G
+
+    # Second install should make only a partial request to /versions
+    bundle! :install, :artifice => "compact_index_partial_update"
```

Comment:
Argh, creating a new `CompactIndex::VersionFile` in the etag_response block was creating a file with a new `created_at` line at the top, accurate to the second. Sometimes the second call was within the same second, sometimes it wasn't. Fixed now, and specs are running.